### ISSUE TITLE
[BUG] Plugin (xz9wYiSRk) broke after update

### ISSUE THREAD

**Issue by kman3107** [HaveAGitGat/Tdarr_Plugins#690]
**Describe the bug**
Plugin xz9wYiSRk(Tdarr_Plugin_henk_Keep_Native_Lang_Plus_Eng) has stopped working after updating Tdarr
`Error: Cannot find module '@cospired/i18n-iso-languages'`

**To Reproduce**
1. Requeue a file that has already passed the flow once
2. It fails

**Expected behavior**
1. Requeue a file that has already passed the flow once
2. It passes

**Please provide the following information:**
- Config files 
[Tdarr_Server_Config.json](https://github.com/user-attachments/files/16576374/Tdarr_Server_Config.json)
[Tdarr_Node_Config.json](https://github.com/user-attachments/files/16576375/Tdarr_Node_Config.json)

- Job reports
Success: [qw0LZB-8YxG-log.txt](https://github.com/user-attachments/files/16576379/qw0LZB-8YxG-log.txt)
Fail: [qjX2rlzOo-log.txt](https://github.com/user-attachments/files/16576380/qjX2rlzOo-log.txt)

- Log files 
Node: [Tdarr_Node_Log.txt](https://github.com/user-attachments/files/16576396/Tdarr_Node_Log.txt)
Server: [Tdarr_Server_Log.txt](https://github.com/user-attachments/files/16576397/Tdarr_Server_Log.txt)

-Worker error
```
[2024-08-12T01:12:42.732] [ERROR] Tdarr_Node - Error: Cannot find module '@cospired/i18n-iso-languages'
Require stack:
- /app/Tdarr_Node/assets/app/plugins/Community/Tdarr_Plugin_henk_Keep_Native_Lang_Plus_Eng.js
- /app/Tdarr_Node/assets/app/plugins/FlowPlugins/FlowHelpers/1.0.0/classicPlugins.js
- /app/Tdarr_Node/assets/app/plugins/FlowPlugins/CommunityFlowPlugins/classic/runClassicTranscodePlugin/1.0.0/index.js
- /app/Tdarr_Node/srcug/workers/getFlowPlugins.js
- /app/Tdarr_Node/srcug/workers/worker1.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1140:15)
    at Module._load (node:internal/modules/cjs/loader:981:27)
    at Module.require (node:internal/modules/cjs/loader:1231:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.plugin (/app/Tdarr_Node/assets/app/plugins/Community/Tdarr_Plugin_henk_Keep_Native_Lang_Plus_Eng.js:261:31)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5){
  "code": "MODULE_NOT_FOUND",
  "requireStack": [
    "/app/Tdarr_Node/assets/app/plugins/Community/Tdarr_Plugin_henk_Keep_Native_Lang_Plus_Eng.js",
    "/app/Tdarr_Node/assets/app/plugins/FlowPlugins/FlowHelpers/1.0.0/classicPlugins.js",
    "/app/Tdarr_Node/assets/app/plugins/FlowPlugins/CommunityFlowPlugins/classic/runClassicTranscodePlugin/1.0.0/index.js",
    "/app/Tdarr_Node/srcug/workers/getFlowPlugins.js",
    "/app/Tdarr_Node/srcug/workers/worker1.js"
  ]
}
```

 - OS Tdarr runs on: Unraid 6.12.11 (Linux)
 - OS Viewing Tdarr from: Windows 11
 - Browser firefox
 - Version 129.0

**Additional context**
EDIT: On docker using this image `https://hub.docker.com/r/haveagitgat/tdarr`
Everything was working just fine. Then I updated from 2.23.01 to 2.24.01 and it stopped working.

PS. Have not tried downgrading yet. Hoping this is an easy fix and I'm not in a rush.

**Comment by theg00s3** [comment]
I'm seeing this issue as well

**Comment by HaveAGitGat** [comment]
Thanks, I've released 2.24.02 which fixes this. It was due to this plugin using scoped package `@cospired/i18n-iso-languages`, the scoped part being the `@cospired` which is slightly different to standard dependency notation such as `axios`. Should now work fine.

```
2.24.02 [12th August 2024]
- Fixed classic plugin error when installing scoped packages affecting Tdarr_Plugin_henk_Keep_Native_Lang_Plus_Eng
```
